packages/service/lib/plugins/metrics.js

```diff
@@ -5,16 +5,24 @@ const http = require('node:http')
 const { eventLoopUtilization } = require('node:perf_hooks').performance
 const fastify = require('fastify')
 const fp = require('fastify-plugin')
+const promClient = require('prom-client')
 
 const metricsPlugin = fp(async function (app, opts = {}) {
-  const defaultMetrics = opts.defaultMetrics ?? { enabled: true }
   const prefix = opts.prefix ?? ''
 
+  if (opts.labels) {
+    const labels = opts.labels ?? {}
+    promClient.register.setDefaultLabels(labels)
```

Comment:
Please import a Register class from the prom-client and create a new register. Otherwise you will modify the global register.

Reply:
Done, PTAL